src/screens/Menu/Menu.js

```diff
@@ -254,17 +255,19 @@ const Menu = ({
         contentContainerStyle={{ width: '100%', padding: spacing.layoutSides, paddingBottom: 40 }}
         ListFooterComponent={
           <Footer>
-            <MigrateEnsBanner style={{ marginTop: 15 }} />
-            {showMigrateWallet && (
-              <FooterBanner>
-                <MigrateWalletBanner onPress={navigateToKeyBasedAssetMigration} />
-              </FooterBanner>
-            )}
+            <BannersWrapper>
+              <MigrateEnsBanner style={styles.footerBanner} />
+
+              <WalletMigrationArchanovaBanner style={styles.footerBanner} />
```

Comment:
Do you think we can add feature flag for Archanova migration too?

Reply:
@IAmKio ^

Reply:
DOne